xls/dev_tools/check_ir_equivalence_main_test.py

```diff
@@ -135,34 +143,39 @@ def _check_equiv(
         ],
         check=False,
         stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
     )
-    return res.returncode == 0, res.stdout.decode("utf8")
+    return res.returncode == 0, res.stdout.decode("utf8"), res.stderr.decode("utf8")
 
   def test_detects_different_functions(self):
-    res, msg = self._check_equiv(ADD_IR, NOT_ADD_IR)
+    res, stdout, stderr = self._check_equiv(ADD_IR, NOT_ADD_IR)
     self.assertFalse(res)
-    self.assertIn("Verified NOT equivalent", msg)
+    self.assertIn("Verified NOT equivalent", stdout)
 
   def test_ignores_assert(self):
-    res, msg = self._check_equiv(PROC_IR_WITH_ASSERT, PROC_IR, act_count=3)
+    res, stdout, stderr = self._check_equiv(PROC_IR_WITH_ASSERT, PROC_IR, act_count=3)
     self.assertTrue(res)
-    self.assertIn("Verified equivalent", msg)
+    self.assertIn("Verified equivalent", stdout)
 
   def test_detects_equiv_functions(self):
-    res, msg = self._check_equiv(ADD_IR, NOT_NOT_ADD_IR)
+    res, stdout, stderr = self._check_equiv(ADD_IR, NOT_NOT_ADD_IR)
     self.assertTrue(res)
-    self.assertIn("Verified equivalent", msg)
+    self.assertIn("Verified equivalent", stdout)
 
   def test_detects_different_procs(self):
-    res, msg = self._check_equiv(PROC_IR, NEG_PROC_IR, act_count=14)
+    res, stdout, stderr = self._check_equiv(PROC_IR, NEG_PROC_IR, act_count=14)
     self.assertFalse(res)
-    self.assertIn("Verified NOT equivalent", msg)
+    self.assertIn("Verified NOT equivalent", stdout)
 
   def test_detects_equiv_procs(self):
-    res, msg = self._check_equiv(PROC_IR, NEG_NEG_PROC_IR, act_count=14)
+    res, stdout, stderr = self._check_equiv(PROC_IR, NEG_NEG_PROC_IR, act_count=14)
     self.assertTrue(res)
-    self.assertIn("Verified equivalent", msg)
+    self.assertIn("Verified equivalent", stdout)
 
+  def test_lhs_has_no_top(self):
+    res, msg, stderr = self._check_equiv(ADD_IR, ADD_NO_TOP_IR)
```

Comment:
Copybara doesn't like all the unused variables (mostly stderr, this last msg is a problem). We can fix it up internally, but you might be able to do it faster. LMK

Reply:
Ah, nevermind lol. Didn't see Alex's comment